python/ql/lib/semmle/python/frameworks/Stdlib.qll

```diff
@@ -3939,6 +3939,176 @@ private module StdlibPrivate {
     }
   }
 
+  // ---------------------------------------------------------------------------
+  // Flow summaries for container methods
+  // ---------------------------------------------------------------------------
+  /** A flow summary for `copy`. */
+  class CopySummary extends SummarizedCallable {
+    CopySummary() { this = "collection.copy" }
+
+    override DataFlow::CallCfgNode getACall() {
```

Comment:
Have you considered having both `getACallSimple` and `getACall`, as in Ruby? Where the former may not depend on API graphs, and hence can be used in the call graph construction/type tracking?

Reply:
Yes, this is added in https://github.com/github/codeql/pull/13178 where our type trackers learn about flow summaries. After that, we should reformulate these summaries to use `getACallSimple`.